libraries/client/preprocessors.py

```diff
@@ -3,20 +3,29 @@
 import re
 from glob import glob
 from shutil import copy
+from libraries.app.app import App
 from libraries.door43_tools.bible_books import BOOK_NUMBERS
 from libraries.general_tools.file_utils import write_file, read_file
 from libraries.resource_container.ResourceContainer import RC
 from libraries.resource_container.ResourceContainer import BIBLE_RESOURCE_TYPES
 
 
-def do_preprocess(rc, repo_dir, output_dir):
+def do_preprocess(rc, repo_dir, output_dir, repo_name=None):
```

Comment:
Already passing rc into here, why not just use rc.repo_name? It will get the repo name from the directory, so if it is for tests, just make sure the repo being tested is extracted to a directory with the repo's name.